packages/orchestration/src/exos/chain-hub.js

```diff
@@ -336,8 +360,53 @@ export const makeChainHub = (agoricNames, vowTools) => {
       // @ts-expect-error XXX generic parameter propagation
       return lookupChainsAndConnection(primaryName, counterName);
     },
+
+    /**
+     * Register an asset that may be held on a chain other than the issuing
+     * chain.
+     *
+     * @param {import('../orchestration-api.js').Denom} denom - on the holding
```

Comment:
this looks to be imported
```suggestion
     * @param {Denom} denom - on the holding
```